docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -119,13 +119,24 @@ Date and time functions
 
 .. function:: at_timezone(timestamp, zone) -> timestamp(p) with time zone
 
-    Change the time zone component of ``timestamp`` with precision ``p`` to
-    ``zone`` while preserving the instant in time.
+    Returns the timestamp specified in ``timestamp`` with time zone converted from the session time zone
+    to the time zone specified in ``zone`` with precision ``p``::
+
+        SELECT current_timezone()
+        -- America/New York
+
+        SELECT at_timezone(TIMESTAMP '2022-11-01 00:00:00', 'America/Los_Angeles')
```

Comment:
Might be better to use a different time than midnight.. also ... maybe explain that there are 3h difference between the two timezones and therefore the time changes..